Source/panels/spell_list.cpp

```diff
@@ -111,6 +133,10 @@ void DrawSpell(const Surface &out)
 	std::optional<std::string_view> hotkeyName = GetHotkeyName(spl, myPlayer._pRSplType, true);
 	if (hotkeyName)
 		PrintSBookHotkey(out, position, *hotkeyName);
+
+	if (*GetOptions().Gameplay.showRemainingCharges) {
+		DrawRemainingCharges(out, position);

```

Comment:
`PrintSBookRemainingCharges()`  would align better with similar functions in this file.